models/home.js

```diff
@@ -0,0 +1,27 @@
+
+var db = require('../data/venues-db');
+
+/*
+ * GET
+ * return the homepage data
+ */
+exports.get = function(callback) {
+	var venues = null;
+
+	// mock request to fetch venues data
+	venues = JSON.parse(JSON.stringify(db));
```

Comment:
Don't think this is necessary, may as well just use the db directly as we're not making modifications to it